engine/swarm/index.md

```diff
@@ -4,17 +4,16 @@ keywords: docker, container, cluster, swarm
 title: Swarm mode overview
 ---
 
-To use Docker Engine in swarm mode, install the Docker Engine `v1.12.0` or
-later from the [Docker releases GitHub
-repository](https://github.com/moby/moby/releases). Alternatively, install
-the latest Docker for Mac or Docker for Windows Beta.
+To use Docker in swarm mode, install Docker `1.12.0` or later. Alternatively,
+install the latest Docker for Mac or Docker for Windows. Install instructions
+for all platforms are [here](https://docs.docker.com/engine/installation/). The
+most recent version is Docker 17.06.
```

Comment:
@thaJeztah I'm not sure what the issue would be with telling people, at time of installation, to install the most up-to-date version of Docker for Mac or Windows. I think this section is needed here, first to make sure they know that only `1.12.0` version of Docker Engine is supported, that if they are on Docker for Mac or Windows, the most up-to-date version will take care of that and they don't need to worry about it, and finally, where to go on the off chance that they are using an earlier version of Docker Engine. 

The only thing that will be time sensitive about the above section is that I'll have to remember to change what "the most recent version .." is, and I was hoping we could figure out a way to automate that. For now, I'm happy to update it, and I think it's nice to have.